docs/Makefile

```diff
@@ -18,3 +27,13 @@ image: ## Build a container image for the preview of the website
 clean:
 	docker image rm $(CONTAINER_IMAGE)
 
+.PHONY: tracing-policy-docs
+tracing-policy-docs: ## Generate Tracing Policy reference page from CRDs.
+	docker run --rm --platform linux/amd64 \
+		-u $(shell id -u):$(shell id -g) \
+		-v ${PWD}:/workdir \
+		-v $(abspath $(CURDIR)/../pkg/k8s/apis/cilium.io/client/crds/v1alpha1):/crds \
+		ghcr.io/fybrik/crdoc:latest \
+		--resources /crds \
```

Comment:
```suggestion
	$(CRDOC) --resources /crds \
```

Reply:
still using `/crds` is not ideal as it wouldn't work when using another `CRDOC` value like a locally installed version. Maybe consider matching the path inside the container with the one outside of the container so that you can reuse `TRACING_POLICY_CRDS_DIR`